package.json

```diff
@@ -9,6 +9,8 @@
     "prestart": "yarn run build:tokens",
     "start": "NODE_OPTIONS=--openssl-legacy-provider start-storybook -p 6006",
     "build": "NODE_OPTIONS=--openssl-legacy-provider build-storybook",
+    "format": "prettier --write \"src/**/*.{ts,tsx}\"",
```

Comment:
Suggestion; can we adopt the prettierrc file from webapp for consistency across GovConnex projects please?

Reply:
yup i have added this :)